src/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const removeRow = document.querySelector('.remove-row');
+const appendRow = document.querySelector('.append-row');
+
+function examination(maxLength, button, maxOrMin) {
```

Comment:
it's not a school test in this function, rename to `toggleDisabled` or something)